src/api/model/integration.test.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright 2021 RelationalAI, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+import {
+  createDatabaseIfNotExists,
+  createEngineIfNotExists,
+  getClient,
+} from '../../testUtils';
+import Client from '../client';
+import { Model } from '../transaction/types';
+
+describe('Integration', () => {
+  const databaseName = `js-sdk-tests-${Date.now()}`;
+  const engineName = `js-sdk-tests-${Date.now()}`;
+  let client: Client;
+
+  jest.setTimeout(1000 * 60 * 10);
+
+  beforeAll(async () => {
+    client = await getClient();
+
+    await createEngineIfNotExists(client, engineName);
+    await createDatabaseIfNotExists(client, databaseName);
+  });
+
+  afterAll(async () => {
+    await client.deleteEngine(engineName);
+    await client.deleteDatabase(databaseName);
+  });
+
+  describe('Models actions', () => {
+    it('should list all models', async () => {
+      const models = await client.listModels(databaseName, engineName);
+      expect(models?.length).toBeGreaterThan(0);
+    });
+
+    it('should install model', async () => {
```

Comment:
Maybe change the name? It tests `installModels`, `getModel`, `listModels`. Separate tests are good too.